### DISCUSSION TITLE
Copying code plugs between radio types.

### DISCUSSION THREAD

**Discussion by vk1kcm** [hmatuschek/qdmr#329] [General]
I have an RT3s with a fairly complete codeplug for the local area.  We were having a group meet tonight to program radios and I was trying to take my codeplug and convert it for an Anytone 878UV.  It verified OK and seemed to write ok, but the Anytone didn't like it and was doing weird things, like not showing the selected channel and transmitting on TG 30008 randomly.

Is converting code plugs like this something that qdmr is supposed to do?  Is there any other way (aside from editing yaml files) to merge code plugs?

Thanks,
Carl.

**Comment by hmatuschek** [comment]
Yes, this should work and should be the one feature qdmr is designed for. However, I need to have some additional information: Which version of qdmr are you running and what are the firmware version of both, the RT3S and D878UV. Also, if there is nothing secret in your codeplugs, could you add it here (the yaml file).

**Reply by vk1kcm** [reply]
Hi Hannes,
I have the yaml file for my rt3s and for the 878's original codeplug I copied from the radio, but not the one I uploaded.  The only difference however was I updated the DMR ID, Radio Name and Intro line 1 to be Peter, VK1PDW's details.  I'll attach both the yaml files I currently have below.

The qdmr I'm using is one I compiled up from a git pull from the 13th April 23.  I've been using that to get the call-sign db loaded into my rt3s.

Firmware versions, 
rt3s - P018.016, which I believe is the latest
878UV - 1.16

I can probably borrow either that radio or another 878UV for testing if that would help.

Carl.
[Archive.zip](https://github.com/hmatuschek/qdmr/files/11284222/Archive.zip)

**Reply by hmatuschek** [reply]
Ok, the firmware of the D878UV is a bit old. The current version is 3.01. I'll have a look, if there is something that triggers a bug in qdmr.

**Reply by allesand** [reply]
@vk1kcm : recent firmware for the 878:

http://www.wouxun.us/category.php?category_id=93

Be sure to get the right version - the "top button color" way works *most* of the time (there are a few units that do not match that description..) Version 3.02n is well-tested meanwhile and works well with qdmr.
